C-Plus-Plus/other/lcm.cpp

```diff
@@ -0,0 +1,55 @@
+//Using GCD, find LCM of two number.
+
```

Comment:
You may add description of the in-built LCM function that was introduced in C++17, which may come handy during competitive programming. You can read about it [here](https://www.geeksforgeeks.org/stdlcm-in-cpp17/)